### PR TITLE
[WIP] Add supporters structure in footer and as a separated page

### PR DESCRIPTION
**Pull request by jonasschnelli**
Groundwork for supporters in footer and as a separated page.
Maybe needs to include some .md files to remove the actual content from the .html template.

@btcdrak interested to continue this?

bildschirmfoto 2016-01-18 um 11 27 51

### CODE REVIEW

**Comment by btcdrak** [comment]
needs a `_posts/en/page/supporters.md` page with the include

File: _includes/_supporters_short.html [hunk 1]
```diff
@@ -0,0 +1,29 @@
+<div class="show-your-support show-your-support-small">
+  <h3>Supported by</h3>
```
Comment (btcdrak) [inline _includes/_supporters_short.html]: `Show your support` with github ticket + tweet link.

**Comment by jonasschnelli** [comment]
Force push update.
Fixed text nits, added correct github link. Added correct twitter autolink.
Made 1 grid size left and right margin.

**Comment by jonasschnelli** [comment]
Currently on the my staging test site: https://coresitetest.jonasschnelli.ch

**Comment by btcdrak** [comment]
This was semi pulled already.

**Comment by MarcoFalke** [comment]
@jonasschnelli Needs merge conflict solved.

**Pull request closed without merging**